src/pybind/mgr/cephadm/module.py

```diff
@@ -2066,28 +2066,58 @@ def blink(host, dev, path):
         return blink(locs)
 
     def get_osd_uuid_map(self, only_up=False):
-        # type: (bool) -> Dict[str,str]
+        # type: (Optional[bool]) -> Dict[str, str]
```

Comment:
`Optional[bool]` looks always a bit odd to me. If possible, I'd keep it as a simple `bool`

Reply:
resolved